integration_test.go

```diff
@@ -191,6 +202,40 @@ func textRoundTrip(t *testing.T, fp string) {
 	}
 }
 
+func testLoadBad(t *testing.T, fp string) {
+	file, er := os.Open(fp)
+	if er != nil {
+		t.Fatal(er)
+	}
+	defer file.Close()
+
+	r := NewReader(file)
+
+	err := testBrokenReader(t, r)
+
+	if r.Err() == nil && err == nil {
+		t.Fatal("Should have failed loading \"" + fp + "\".")
+	} else {
+		fmt.Println("expectedly failed loading " + r.Err().Error())
```

Comment:
Should we be using print statements?

Reply:
If we get to ``else``, it means the test has passed, and it just provides a message.
I can replace ``print`` with ``t.Log()`` - it wouldn't change the behaviour.

Reply:
Thats fine, it just looks weird with print statements